SA3/qa_web/templates/qa_web/answerspage.html

```diff
@@ -40,16 +40,64 @@
     </style>
   </head>
   <body>
-  <h1>{{ currentQuestion.title }}</h1>
-  <p>{{ currentQuestion.content }}</p>
-  <h2>Question by {{ currentQuestion.owner }}</h2>
-    <div id="wmd-button-bar"></div>
-    <textarea id="wmd-input" class="wmd-input"></textarea>
-    <div id="wmd-preview" class="wmd-panel wmd-preview"></div>
-  <script>
-      var converter = Markdown.getSanitizingConverter();
-      var editor = new Markdown.Editor(converter);
-      editor.run();
-    </script>
+  <!-- Display the question's data -->
+  <div id="question">
+      <h1>{{ currentQuestion.title }}</h1>
+      <p>{{ currentQuestion.content }}</p>
+      <h2>Question by {{ currentQuestion.owner }} on {{ currentQuestion.creation_date }}</h2>
+      <hr>
+  </div>
+  <div id="answers">
+  <!-- Display the answers -->
+  {% for answer in bestAnswer %}
```

Comment:
Why does the template iterate over bestAnswer if there should only be 1 best answer per question?

Reply:
For this, pass only one dictionary value answer in your dictionary context in `views.py`.